demo/OrderCloud.DemoWebApi/AppSettings.cs

```diff
@@ -24,4 +22,12 @@ public class EnvironmentSettings
 	{
 		public string BuildNumber { get; set; }
 	}
+
+	public class OrderCloudSettings
+	{
+		public string ApiUrl { get; set; }
+		public string ClientID { get; set; }
+		public string ClientSecret { get; set; }
```

Comment:
Can you change this to MiddlewareClientID and MiddlewareClientSecret? Its possible users will add other clientIDs down the road and this gives room for that

Reply:
Done!